Makefile.Common

```diff
@@ -19,6 +19,7 @@ LINT=golangci-lint
 IMPI=impi
 # BUILD_TYPE should be one of (dev, release).
 BUILD_TYPE?=release
+RUNNING_ON_GITHUB_ACTION=$(GITHUB_ACTIONS)
```

Comment:
There's still something odd here. When running on the main branch, I see the test summary but I don't see it when this PR is checked out:

```
$ make test
go test -race -timeout 300s --tags="" ./...
ok  	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter	0.063s
ok  	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter/internal/tenant	0.032s
$ gh pr checkout 16427
Switched to branch 'added-foresight'
$ make test
$
```

Reply:
Hi @jpkrohling,

There was missing `\` in the `Makefile.common`. Could you please re-pull `added-foresight` branch and re-run?

Reply:
I've got the same result. Did you get it working locally? I'm trying it out in the Loki Exporter directory.

Reply:
Ok, sorry for the issue. Let us to reproduce on our local

Reply:
@jpkrohling, we have identified the issue. Working on the fix. FYI

Reply:
@jpkrohling, we have pushed a fix (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16427/commits/b8b62a8743d0268f8a57c998a5154a56151123b7) for silent mode in the Makefile.common. Could you please try again by re-pulling from the branch?

Reply:
Looks better, but the output is still different than the one I get from `main`. Looks like it's verbose now?

```
$ make test
if [ "" = "true" ]; then \
	go test -race -v -timeout 300s --tags="" ./... -json > ./foresight-test-report.json; \
else \
	go test -race -v -timeout 300s --tags="" ./...; \
fi
=== RUN   TestLoadConfigNewExporter
=== PAUSE TestLoadConfigNewExporter
=== RUN   TestIsLegacy
=== RUN   TestIsLegacy/not_legacy
=== RUN   TestIsLegacy/format_is_set_to_body
=== RUN   TestIsLegacy/a_label_is_specified
=== RUN   TestIsLegacy/a_tenant_is_specified
=== RUN   TestIsLegacy/a_tenant_ID_is_specified
--- PASS: TestIsLegacy (0.00s)
    --- PASS: TestIsLegacy/not_legacy (0.00s)
    --- PASS: TestIsLegacy/format_is_set_to_body (0.00s)
    --- PASS: TestIsLegacy/a_label_is_specified (0.00s)
    --- PASS: TestIsLegacy/a_tenant_is_specified (0.00s)
    --- PASS: TestIsLegacy/a_tenant_ID_is_specified (0.00s)
=== RUN   TestLoadConfig
=== PAUSE TestLoadConfig
...
...
```

vs.

```
$ make test
go test -race -timeout 300s --tags="" ./...
ok  	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter	(cached)
ok  	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter/internal/tenant	(cached)
```

Reply:
Ok, because verbose mode is disabled 3 days ago: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16416/files

But we have not rebased from master. Let me rebase from upstream master then, it should be OK.

Reply:
@jpkrohling, Ok, just rebased from upstream master. So verbose mode is disabled for tests. Could you please try again?

Reply:
Looks good now!